api/app/tests/Integration/Model/DeputyshipProcessingRawDbAccessIntegrationTest.php

```diff
@@ -64,7 +64,7 @@ public function testInsertOrderAndFindOrderId(): void
         $this->sut->endTransaction();
 
         // use SUT to find the ID of the order just inserted
-        $orderId = $this->sut->findOrderId($uid);
+        $orderId = $this->sut->findOrderId($uid)->data;
```

Comment:
I'm not sure how this test is passing atm, but probably because the client exists.